src/search/FileFilters.js

```diff
@@ -337,10 +344,24 @@ define(function (require, exports, module) {
             });
         }
 
+        $nameField.bind('input', function () {
+            var remainingCharacters = FILTER_NAME_CHARACTER_MAX - $(this).val().length;
+            $remainingField.text(StringUtils.format(
+                Strings.FILTER_NAME_REMAINING,
+                remainingCharacters
+            ));
+
+            if (remainingCharacters < 0) {
+               $remainingField.addClass("exclusions-name-characters-limit-reached");
+            } else {
+               $remainingField.removeClass("exclusions-name-characters-limit-reached");
+            }
+        });
+
         dialog.done(function (buttonId) {
             if (buttonId === Dialogs.DIALOG_BTN_OK) {
                 // Update saved filter preference
-                setActiveFilter({ name: $nameField.val(), patterns: getValue() }, index);
+                setActiveFilter({ name: $nameField.val().substr(0, FILTER_NAME_CHARACTER_MAX), patterns: getValue() }, index);
```

Comment:
do not do this, `OK` button should be disabled if `$nameField.val()` is more than `FILTER_NAME_CHARACTER_MAX`